tests/saw/spec/handshake/rfc_handshake_tls12.cry

```diff
@@ -1,5 +1,8 @@
 // This module provides an specification of the TLS 1.2 handshake message
 // state machine according to RFCs 5246, 5077 and 6066 (OCSP extension only).
+// The NPN extension is based on the draft
+// https://datatracker.ietf.org/doc/html/draft-agl-tls-nextprotoneg-04#ref-spdy
```

Comment:
Unfortunately, openssl didn't implement the latest draft, so we didn't either to be interoperable :) This link should be https://datatracker.ietf.org/doc/html/draft-agl-tls-nextprotoneg-03

Reply:
Good catch. See most recent commit.